packages/opentelemetry-node/lib/elastic-node-sdk.js

```diff
@@ -89,6 +119,23 @@ class ElasticNodeSDK extends NodeSDK {
         const metricsDisabled =
             process.env.ELASTIC_OTEL_METRICS_DISABLED === 'true';
         if (!metricsDisabled) {
+            // Default logs exporter based on environment. Using `http/proto` as default protocol
+            const metricsExporters = {
+                grpc: OTLPMetricExporterGrpc,
+                'http/json': OTLPMetricExporterHttp,
+                'http/protobuf': OTLPMetricExporterProto,
+            };
+            const metricsExportProtocol =
+                process.env.OTEL_EXPORTER_OTLP_METRICS_PROTOCOL ||
+                process.env.OTEL_EXPORTER_OTLP_PROTOCOL ||
+                'http/protobuf';
+            let OTLPMetricExporter = metricsExporters[metricsExportProtocol];
+            if (!OTLPMetricExporter) {
+                log.debug(
```

Comment:
What do you think about a log.warn level message when there is a bogus config var that is being ignored?
See https://github.com/elastic/apm/blob/main/specs/agents/configuration.md#invalid-configuration-values

What level does the vanilla sdk-node use for these envvars?

Reply:
I think you're right to elevate this to a warn.

> What level does the vanilla sdk-node use for these envvars?

Just checked and it also logs a waning message. An example https://github.com/open-telemetry/opentelemetry-js/blob/c9ea5f24286bdbd6a2ecd465ca3696d4bbdfb941/experimental/packages/opentelemetry-sdk-node/src/TracerProviderWithEnvExporter.ts#L52

So I'm changing it to `log.warn` :)